templates/gitlab.rb.j2

```diff
@@ -33,11 +28,26 @@ gitlab_rails['ldap_base'] = '{{ gitlab_ldap_base }}'
 
 # GitLab Nginx
 ## See https://gitlab.com/gitlab-org/omnibus-gitlab/blob/master/doc/settings/nginx.md
+nginx['listen_https'] = {{ 'true' if gitlab_nginx_enabled == true else 'false' }}
```

Comment:
This seems to be conflating two options—this would not allow you to use nginx but have ssl turned off...